doc/cephfs/client-config-ref.rst

```diff
@@ -4,115 +4,115 @@
+:Description: Check if the reply from the MDS contains required capabilities.
 :Type: Boolean
 :Default: ``false``
 
 ``client_dirsize_rbytes``
 
-:Description: Use recursive size of directory for directory ``st_size`` stat member.
+:Description: If set to `true`, use the recursive size of a directory (that is, total of all descendants).
 :Type: Boolean
 :Default: ``true``
 
 ``client_max_inline_size``
 
-:Description: Maximum size of inlined data stored in a file inode (rather than in a separate data object in RADOS). This setting only applies if the ``inline_data`` flag is set on the MDSMap.
+:Description: Set the maximum size of inlined data stored in a file inode rather than in a separate data object in RADOS. This setting only applies if the ``inline_data`` flag is set on the MDS map.
 :Type: Integer
 :Default: ``4096``
 
 ``client_metadata``
 
 :Description: Comma-delimited strings for client metadata sent to each MDS.
 :Type: String
-:Default: ``""`` (N/A)
+:Default: ``""`` (empty string)
 
 ``client_mount_gid``
 
-:Description: Group-id of CephFS mount.
+:Description: Set the group ID of CephFS mount.
 :Type: Integer
 :Default: ``-1``
 
 ``client_mount_timeout``
 
-:Description: Timeout for CephFS mount.
+:Description: Set the timeout for CephFS mount in seconds.
 :Type: Float
-:Default: ``300.0`` (seconds)
+:Default: ``300.0``
 
 ``client_mount_uid``
 
-:Description: User-id of CephFS mount.
+:Description: Set the user ID of CephFS mount.
 :Type: Integer
 :Default: ``-1``
 
 ``client_mountpoint``
 
-:Description: Directory to mount on the CephFS file system.
+:Description: Directory to mount on the CephFS file system. An alternative to the `-r` option of the `ceph-fuse` command.
 :Type: String
 :Default: ``"/"``
 
 ``client_oc``
 
-:Description: Enable object caching. **Setting currently ignored.**
+:Description: Enable object caching.
 :Type: Boolean
 :Default: ``true``
 
 ``client_oc_max_dirty``
 
-:Description: Maximum number of dirty bytes in object cache.
+:Description: Set the maximum number of dirty bytes in the object cache.
 :Type: Integer
 :Default: ``104857600`` (100MB)
 
 ``client_oc_max_dirty_age``
 
-:Description: Maximum age of dirty data in the object cache before writeback.
+:Description: Set the maximum age in seconds of dirty data in the object cache before writeback.
 :Type: Float
 :Default: ``5.0`` (seconds)
 
 ``client_oc_max_objects``
 
-:Description: Maximum number of objects in the object cache.
+:Description: Set the maximum number of objects in the object cache.
 :Type: Integer
 :Default: ``1000``
 
 ``client_oc_size``
 
-:Description: Maximum size of cached data in the object cache.
+:Description: Set how many megabytes of data will the client cache.
 :Type: Integer
 :Default: ``209715200`` (200MB)
 
 ``client_oc_target_dirty``
 
-:Description: Target size of dirty data. It is recommended to keep this low.
+:Description: Set the target size of corrupted data. We recommend to keep this number low.
```

Comment:
Oh boy :) I had the same issue also in the downstream docs, so thank you for double checking. 

This should be fixed with: https://github.com/ceph/ceph/pull/13065/commits/80db40f8559128baadad42b925ae813e51a31409